go.mod

```diff
@@ -3,11 +3,13 @@ module github.com/openshift/generic-admission-server
 go 1.13
 
 require (
-	github.com/spf13/cobra v0.0.5
-	k8s.io/api v0.19.5
-	k8s.io/apimachinery v0.19.5
-	k8s.io/apiserver v0.19.0
-	k8s.io/client-go v0.19.5
-	k8s.io/component-base v0.19.5
-	k8s.io/klog/v2 v2.3.0
+	github.com/fsnotify/fsnotify v1.4.9 // indirect
+	github.com/openshift/library-go v0.0.0-20210407140145-f831e911c638 // indirect
+	github.com/spf13/cobra v1.1.1
+	k8s.io/api v0.21.0-rc.0
```

Comment:
why not `v1.21.0` ?

Reply:
or even `1.22` ?

Reply:
Which repo specifically?

Reply:
we usually pin all the repos to the same level - so I'd change all k8s deps to `1.22`

Reply:
Sure, I updated all repos to v0.22.0.